cmd/link/link.go

```diff
@@ -3,20 +3,32 @@ package link
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/fs/config/flags"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/spf13/cobra"
 )
 
+var (
+	expire = time.Minute * 15
+	unlink = false
+)
+
 func init() {
 	cmd.Root.AddCommand(commandDefinition)
+	cmdFlags := commandDefinition.Flags()
+	flags.DurationVarP(cmdFlags, &expire, "expire", "", expire, "the amount of time that the URL is valid")
+	flags.BoolVarP(cmdFlags, &unlink, "unlink", "", unlink, "Remove existing public link to file/folder with link command rather than creating")
 }
 
 var commandDefinition = &cobra.Command{
 	Use:   "link remote:path",
 	Short: `Generate public link to file/folder.`,
 	Long: `
+If you supply the --expire flag, it will set expiration time else it will use default 15 minutes.
```

Comment:
This needs a "Note: not all backends support this flag - if the backend doesn't support it then the link returned won't expire."